phpunit.xml.dist

```diff
@@ -1,14 +1,22 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <phpunit
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
-    bootstrap="vendor/autoload.php"
-    colors="true">
-    <coverage processUncoveredFiles="true">
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
+        bootstrap="vendor/autoload.php"
+        cacheDirectory=".phpunit.cache"
+        displayDetailsOnIncompleteTests="true"
+        displayDetailsOnSkippedTests="true"
+        displayDetailsOnTestsThatTriggerDeprecations="true"
+        displayDetailsOnTestsThatTriggerErrors="true"
+        displayDetailsOnTestsThatTriggerNotices="true"
+        displayDetailsOnTestsThatTriggerWarnings="true"
+        colors="true"
```

Comment:
Do you think that `failOnWarning`, `failOnNotice` and friends would be too strict?

Reply:
I don't think this controls whether phpunit fails on warnings. This makes phpunit report tests that produced warnings or notices in test results.

Reply:
Sorry - I meant: Do you think that `failOnWarning` and friends should be *added* to PHPUnit's config?

Reply:
They should.
Not the decision I am going to make here. *We* don't use them. Users can tighten configuration themselves.